.travis.yml

```diff
@@ -11,8 +11,6 @@ before_script:
   - yarn run ganache > /dev/null &
 script:
   - yarn lint
-  - yarn solhint
```

Comment:
I know you mentioned something about this in the description, but I don't see a replacement for this here. Is it already somewhere else?

Reply:
https://github.com/gnosis/dex-contracts/pull/741/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R24-R30